jython_cli.java

```diff
@@ -19,6 +19,8 @@
 public class jython_cli {
 
     // Increment this number with each new release of jython-cli
+    // But if the version number of Jython changes (for example 2.7.4 becomes 2.7.5), then
+    // the FIX_NUMBER is reset to 0 again.
```

Comment:
Please add one more sentence describing the role of this number for the catalog version, like:
```
// This number appended to the Jython version forms the version of jython-cli as
// [Jython version].[FIX_NUMBER], e.g. 2.7.4.0
```

Reply:
Fixed. Thanks.